data_steward/gcloud/bq/__init__.py

```diff
@@ -478,3 +484,44 @@ def table_exists(self, table_id: str, dataset_id: str = None) -> bool:
             return True
         except NotFound:
             return False
+
+    def serialize_jobs(self, job_list: list):
+        """
+        Waits on jobs until completion one by one
+
+        :param job_list: list of job_ids
+        """
+        for job_id in job_list:
+            job_info = self.get_job(job_id)
+            if job_info.state != 'DONE':
+                logging.info(f"Waiting on job {job_id} to complete")
+                job_info.result()
```

Comment:
where is this used?

Reply:
This isn't used anywhere but can be used to serialize jobs if a certain order is required.